src/Components/MovieTile/MovieTile.jsx

```diff
@@ -0,0 +1,18 @@
+import styles from './MovieTile.module.css';
+
+export const MovieTile = ({ imageUrl, movieName, releaseYear, relevantGenres, onClick }) => {
+  const handleClick = () => {
+    onClick();
+  };
+
+  return (
+    <div className={styles.container} onClick={handleClick}>
+      <img src={imageUrl} alt={`${movieName}`} />
```

Comment:
Same as above:

```suggestion
      <img src={imageUrl} alt={movieName} />
```

But I don't think alt is needed here. It can be set to an empty string in cases like this:

```suggestion
      <img src={imageUrl} alt="" />
```

When you set a movie title as image's `alt` and there is the same text with this movie title next to the image accessibility checkers will mark it as a warning because they will consider `alt` like this excessive.